vault.js

```diff
@@ -1,4 +1,17 @@
 'use strict';
 module.exports = function() {
-  
+    var fart={};
```

Comment:
Use meaningful variable name. For example it would be reasonable to name this var `vault`.